src/app/Http/Controllers/Admin/ArticleAdminController.php

```diff
@@ -45,9 +45,9 @@ public function store(ArticleCreateRequest $request): Factory|View|Application|R
         return redirect(route('articles.show', [$article]));
     }
 
-    public function show(): Factory|View|Application|RedirectResponse
+    public function show(Article $article): Factory|View|Application|RedirectResponse
     {
-        return true;
+        return redirect(route('articles.show', [$article]));
```

Comment:
It should return a view that shows the article.
It can be done with `return view('pages.admin.articles.show', ['article' => $article]);`